zapier/introduction/zaps-ii/zapier-update-dynamically.md

```diff
@@ -0,0 +1,39 @@
+---
+author: kapnobatai136
+
+category: must-know
+
+aspects:
+  - introduction
+
+type: normal
+
+---
+
+# Update Dynamically
+
+---
+## Content
+
+Now that we have added a new column to our spreadsheet, we also want to populate it. With Zapier, you can search search for data, create new data or dynamically (on the go) update data.
+
+Going back to our zap, let's add another step right at the end. It will also be a Google Sheets step, but this time we want to choose `'Update Spreadsheet Row'`.
```

Comment:
maybe first say why we want to add that step. essentially to update the value in the previously created column, right?